iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Network.java

```diff
@@ -410,6 +411,16 @@ interface BusBreakerView {
          */
         Stream<Bus> getBusStream();
 
+        /**
+         * Get the bus count.
+         * <p>
+         * Depends on the working variant.
+         * @see VariantManager
+         */
+        default int getBusCount() {
+            return Ints.checkedCast(getBusStream().count());
+        }
```

Comment:
To be consistent with the other `getXxxCount()` methods in this interface, we should not propose a default implementation. This way the existing Network's implementation's developers will be aware of the existence of this new method and could provide a more performant implementation.